README.md

```diff
@@ -27,8 +27,15 @@ Optuna can dynamically construct the search spaces for the hyperparameters.
 Optuna has modern functionalities as follows:
 
 - [Lightweight, versatile, and platform agnostic architecture](https://optuna.readthedocs.io/en/stable/tutorial/first.html)
-- [Parallel distributed optimization](https://optuna.readthedocs.io/en/stable/tutorial/distributed.html)
-- [Pruning of unpromising trials](https://optuna.readthedocs.io/en/stable/tutorial/pruning.html)
+  - Handle a wide variety of tasks, while simple installation with few requirements. 
```

Comment:
Just a nit but "while simple installation" sounded a bit unnatural. What do you think of the following?

```suggestion
  - Handle a wide variety of tasks with a simple installation that has few requirements. 
```